packages/aws-cdk/test/integ/cli/cdk-helpers.ts

```diff
@@ -178,7 +178,7 @@ export class TestFixture {
   }
 
   public async cdk(args: string[], options: CdkCliOptions = {}) {
-    return this.shell(['cdk', ...args], {
+    return this.shell(['cdk', '-v', ...args], {
```

Comment:
@rix0rrr To see these logs in the regression suites for the current release this needs to be patched to the previous tests as well..